arch/arm/src/armv7-a/smp.h

```diff
@@ -58,7 +58,7 @@
 #define SMP_STACK_MASK       7
 #define SMP_STACK_SIZE       ((CONFIG_SMP_IDLETHREAD_STACKSIZE + 7) & ~7)
 #define SMP_STACK_WORDS      (SMP_STACK_SIZE >> 2)
-#define SMP_STACK_TOP        (SMP_STACK_SIZE - 8)
+#define SMP_STACK_TOP        (SMP_STACK_SIZE)
```

Comment:
let's replace all SMP_STACK_TOP with SMP_STACK_SIZE?

Reply:
@xiaoxiang781216 
Thanks for the advice.
OK, to reduce complexity, I will replace them and remove SMP_STACK_TOP.

Reply:
Done